src/lighteval/metrics/metrics_sample.py

```diff
@@ -28,6 +28,7 @@
 
 import nltk
 import numpy as np
+import openai
```

Comment:
this import needs to be conditional

Reply:
(with an error message in the import checker, like the other ones)

Reply:
Mhh I don't think we should since `llm-as-judge` is an official metric

Reply:
so I added `openai` in the requirement list

Reply:
I'm not fond of imposing this to all users

Reply:
+ you now have two imports for open ai ^^

Reply:
yep fixing that. will come back to this once the PR to use transformers as llm_judge is merged. I'm not a fan of adding code complexity to avoid a dependency

Reply:
I really think it goes against the spirit of open source to force people to import an openai dep

Reply:
Agreed on that point, a user opened a PR to load `openai` lazily. will be using their PR instead of this one. the only issue is that if we do not have openai installed, the eval will fail after having ran the evaluation for the task.

Reply:
agreed, changed the system